### PR TITLE
Fabo/ledger signout

### LINKED ISSUE #4828: ledger issues
reported by a user

- you can't use multiple ledgers with lunie right now because the sign out is missing
- app.lunie.io/ledger takes a long time to load 
- app.lunie.io/ledger doesn't behave as expected if you're already signed in with a ledger address

### PR DESCRIPTION
**Pull request by faboweb**
Closes #4828

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f53def91e2b2f6ca5b6768f--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
The extension change is merely a lint. No changelog needed, I believe

**Comment by Bitcoinera** [comment]
Why are other comments appearing here?

File: app/src/components/account/UserMenu.vue [hunk 1]
```diff
@@ -166,7 +166,9 @@ export default {
           )
           .map((address) => ({
             ...address,
-            sessionType: address.type || "explore",
+            sessionType: address.sessionType
```
Comment (faboweb) [inline app/src/components/account/UserMenu.vue:169]: for which case did this not work? is this now ever "explore"?

Comment (Bitcoinera) [reply app/src/components/account/UserMenu.vue:169]: no, it was ever "Explore" before. The problem is that it wasn't considering when `address` already has `addressType` as a field

File: app/src/components/account/UserMenu.vue [hunk 2]
```diff
@@ -166,7 +166,9 @@ export default {
...
+              ? address.sessionType
+              : address.type || "explore",
```
Comment (faboweb) [inline app/src/components/account/UserMenu.vue:171]:
```suggestion
              : "explore",
```

File: app/src/components/account/UserMenu.vue [hunk 3]
```diff
@@ -168,7 +168,7 @@ export default {
             ...address,
             sessionType: address.sessionType
               ? address.sessionType
-              : address.type || "explore",
+              : "explore",
```
Comment (Bitcoinera) [inline app/src/components/account/UserMenu.vue:171]: I think some address have a `type` field instead of a `sessionType`. Maybe only legacy addresses

**Review by Bitcoinera — APPROVED** [review]

File: app/src/components/account/UserMenu.vue (hunk 3 above)
Comment (faboweb) [reply app/src/components/account/UserMenu.vue:171]: no, I map them all to sessionType

**Comment by faboweb** [comment]
e2e actually failed...

**Pull request merged**